### DISCUSSION TITLE
ORCiD will not import PubPub DOI

### DISCUSSION THREAD

**Discussion by BruceCaron** [knowledgefutures/pubpub#1343] [Support]
ORCiD spits out an error when I try to add a publication using the PubPub DOI... see image.
<img width="711" alt="pubPUB_orcid_DOI_problem" src="https://user-images.githubusercontent.com/3869816/113777259-b2688600-96df-11eb-8c4a-7e1d07b04c4a.png">

**Comment by gabestein** [comment]
I am able to reproduce, but unfortunately, I don't think it's on our end. The DOI functions just fine, and the metadata looks good in Crossref. I think you'll need to ask ORCiD for help here.
